lib/simple_webdriver_client.ts

```diff
@@ -0,0 +1,65 @@
+import * as http from 'http';
+import * as url from 'url';
+
+export class SimpleWebDriverClient {
```

Comment:
Should have a JSDoc comment block about `SimpleWebDriverClient`. Might also want to consider writing comment blocks over other public methods.